src/bin/pg_autoctl/cli_formation.c

```diff
@@ -61,11 +61,14 @@ keeper_cli_formation_getopts(int argc, char **argv)
 {
 	FormationConfig options = { 0 };
 	int c = 0, option_index = 0, errors = 0;
+	int verboseCount = 0;
 
 	static struct option long_options[] = {
 		{ "pgdata", required_argument, NULL, 'D' },
 		{ "formation", required_argument, NULL, 'f' },
-		{ "help", no_argument, NULL, 'h' },
+		{ "version", no_argument, NULL, 'V' },
```

Comment:
we are having very similar looking functions.  We would need to consolidate them at some time